lcls-twincat-motion/Library/DUTs/DUT_MotionStage.TcDUT

```diff
@@ -1,59 +1,224 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<TcPlcObject Version="1.1.0.1" ProductVersion="3.1.4022.12">
+<TcPlcObject Version="1.1.0.1" ProductVersion="3.1.4022.18">
   <DUT Name="DUT_MotionStage" Id="{10b5775d-5646-4d55-ad77-40abc3888de3}">
     <Declaration><![CDATA[TYPE DUT_MotionStage :
 STRUCT
-	// Hardware
+	(* Hardware *)
+	
+	// PLC Axis Reference
 	Axis: AXIS_REF;
+	// NC Forward Limit Switch: TRUE if ok to move
+	{attribute 'pytmc': '
+		pv: IO:ENA:FWD
+		io: i
+	'}
 	bLimitForwardEnable AT %I*: BOOL;
+	// NC Backward Limit Switch: TRUE if ok to move
+	{attribute 'pytmc': '
+		pv: IO:ENA:BWD
+		io: i
+	'}
 	bLimitBackwardEnable AT %I*: BOOL;
+	// NO Home Switch: TRUE if at home
+	{attribute 'pytmc': '
+		pv: IO:HOME
+		io: i
+	'}
 	bHome AT %I*: BOOL;
-
+	// NC Brake Output: TRUE to release brake
+	{attribute 'pytmc': '
+		pv: IO:BRAKE
+		io: i
+	'}
 	bBrakeRelease AT %Q*: BOOL;
-	
-	// Psuedo-hardware
-	bAllForwardEnable: BOOL:=FALSE; // Forward enable EPS summary
-	bAllBackwardEnable: BOOL:=FALSE; // Forward enable EPS summary
-	bHardwareEnable AT %I*: BOOL; // True when Emergency stop button is not activated
-	
-	bAllEnable: BOOL:=FALSE; // Enable EPS summary encapsulating emergency stop button and any additional motion preventive hardware
+	// NC STO Input: TRUE if ok to move
+	{attribute 'pytmc': '
+		pv: IO:STO
+		io: i
+	'}
+	bHardwareEnable AT %I*: BOOL;
 
-	bGantryForwardEnable: BOOL:=FALSE; // Forward virtual gantry limit switch
-	bGantryBackwardEnable: BOOL:=FALSE; // Reverse virtual gantry limit switch
+	(* Psuedo-hardware *)
+	
+	// Forward enable EPS summary
+	{attribute 'pytmc': '
+		pv: EPS:ENA:FWD
+		io: i
+	'}
+	bAllForwardEnable: BOOL:=FALSE;
+	// Backward enable EPS summary
+	{attribute 'pytmc': '
+		pv: EPS:ENA:BWD
+		io: i
+	'}
+	bAllBackwardEnable: BOOL:=FALSE; 
+	// Enable EPS summary encapsulating emergency stop button and any additional motion preventive hardware
+	{attribute 'pytmc': '
+		pv: EPS:ENA:AMP
+		io: i
+	'}
+	bAllEnable: BOOL:=FALSE;
+	// Forward virtual gantry limit switch
+	{attribute 'pytmc': '
+		pv: GANTRY:ENA:FWD
+		io: i
+	'}
+	bGantryForwardEnable: BOOL:=FALSE;
+	// Backward virtual gantry limit switch
+	{attribute 'pytmc': '
+		pv: GANTRY:ENA:BWD
+		io: i
+	'}
+	bGantryBackwardEnable: BOOL:=FALSE;
 	
-	// Settings
+	(* Settings *)
+	// Change to FALSE if MC_POWER is somewhere else in the project
+	{attribute 'pytmc': '
+		pv: SETTING:bPowerSelf
```

Comment:
I like that these reflect the actual variable names in the PLC program.

Reply:
I'm wondering if I should go back through and match them all up to the PLC variables with no effort put in to make "normal looking" PVs. In some sense these are all "debug" PVs that don't need to be user-facing in any capacity, so making it obvious where they match up inside the PLC is a positive. In my first run-through I was imagining using some of these in a user-facing way (not that users see PV names... usually)

Reply:
Generalizing like this sounds good to me: 
* Categorize with a prefix (`Setting:`, for example, here)
* Use the variable name as-is for the final portion

Reply:
I agree, I will change it as you describe.